### ISSUE TITLE
can not load .pb file 

### ISSUE THREAD

**Issue by Kevin35Day** [JoshuaPiinRueyPan/ViolenceDetection#4]
HI,

I met this error:
Unable to open table file /home/xxxx/ViolenceDetection-master/data/pretrainModels/darknet19/darknet19.pb: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

My Tensorflow is 1.7.0, why can not I load this .pb file?
Can you give me some suggestions?
THANKS!

**Comment by JoshuaPiinRueyPan** [comment]
Hi,
  Did you follow the Step-5 in the Quick Start section in README?  This work is base on the pre-trained Darknet19.  If you did not download the pre-trained model and place it in the specified place, such error will be raised.
